samples/apps/copilot-chat-app/webapi/README.md

```diff
@@ -46,7 +46,9 @@ You can start the WebApi service using the command-line, Visual Studio Code, or
    ```
 
 ## Visual Studio Code
```

Comment:
Does the dev have to open a certain folder (e.g., repo root) to have the commands available to them? If the dev open .../samples/copilot-chat-app/webapi/, will the tasks still kick in in VSCode? If not, we might want to guide the user to open the right folder in VSCode.

Reply:
Yeah I agree the story here could be improved (my solution would be moving samples to their own repo -- a tad drastic). 

As it is now, the README points folks to the solution file in the kernel source path -- so that coupling to the root is basically there. 

We can also add scoped version of the file to this folder and it'd work automatically. If folks are okay with that, I'd gladly do so.

Reply:
Added for now! :) Testing appreciated.